nixos/modules/services/security/munge.nix

```diff
@@ -44,14 +52,20 @@ in
       preStart = ''
         chmod 0700 ${cfg.password}
         mkdir -p /var/lib/munge -m 0711
+        chown munge:munge /var/lib/munge
```

Comment:
maybe ```chown -R``` to change ownership of all files under these directories (especially on upgrade, when the daemon running root will be stopped and restarted under an unprivileged user)?

Reply:
Good point.